great_expectations/datasource/batch_kwargs_generator/table_batch_kwargs_generator.py

```diff
@@ -119,12 +119,16 @@ def _get_iterator(self, generator_asset, query_parameters=None, limit=None, offs
             split_generator_asset = generator_asset.split(".")
             if len(split_generator_asset) == 2:
                 schema_name = split_generator_asset[0]
-                table_name = split_generator_asset[1]
+                if self.engine.dialect.name.lower() == "bigquery":
```

Comment:
We are somewhat inconsistent in checking between using the dialect name and the class of the dialect.

I would like to be consistent where possible, but I think there *is* a legitimate difference in checking which specific driver they're using in some cases vs. checking what the dialect name is set to.

For now, this is an FYI note.